frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -132,55 +130,34 @@ function AdminDropdown() {
 }
 
 /**
- * View Projects navigation menu link.
+ * Disposal Projects navigation dropdown menu.
  */
-function ViewProjects() {
+function DisposeProjectsDropdown() {
   const history = useHistory();
   const keycloak = useKeycloakWrapper();
   return keycloak.hasClaim('admin-properties') && keycloak.hasClaim('admin-projects') ? (
```

Comment:
I don't think this is correct anymore. previously a user would only need the DISPOSE_APPROVE claim to view project approval requests. However, it looks like you've moved the approval requests item to this dropdown, which requires either the admin-properties or admin-projects claim.

Reply:
I'll fix that